wifiPassword/wifiPassword.py

```diff
@@ -62,7 +60,8 @@ def getprofile(self):
         if self.system == "linux" or self.system == "linux2":
             command = "iwgetid -r"
         elif self.system == "darwin":
-            command = ""
+            command = "/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport" + \
```

Comment:
Can you fix the length issue, flake8 results are failing.

Reply:
Sure, give me a sec